README.md

```diff
@@ -1,93 +1,24 @@
-
-## How to run the sample
-
-To run this sample and verify that files are downloaded from primary storage:
-
-1. Add your storage account credentials to the `.env.example` file and then rename it to `.env`.
-
-    ```
-    AZURE_STORAGE_ACCOUNT_NAME=<replace with your storage account name>
-    AZURE_STORAGE_ACCOUNT_ACCESS_KEY=<replace with your storage account access key>
-    ```
-
-    You can find this information in the Azure portal by navigating to your storage account and selecting **Access keys** in the **Settings** section. 
-
-2. Install the required dependencies in the console window by entering `npm install`.
-
-3. Launch the sample from the console window by entering `node index.js`.
-
-4. Wait until the sample reports that the container and file have replicated to secondary storage.
-
-5. When prompted, press D to download the sample file and verify that it comes from primary storage.
-
-To test secondary storage access:
-
-1. Open a command prompt with administrator privileges.
+* Create a storage account.
+* Create a container.
+* Upload a file to blockblob.
+* Enter D to download the file or Q to quit
 
-2. Get the IP address of your storage account primary endpoint domain by entering the following command, replacing `STORAGEACCOUNTNAME` with the name of your storage account.
-
-    ```
-    nslookup STORAGEACCOUNTNAME.blob.core.windows.net
-    ```
-
-3. Get the IP address of your local host by entering `ifconfig` on Linux or `ipconfig` on Windows.
-
-4. Add an invalid static route for a destination host. Replace `<destination_ip>` with your storage account IP address, and `<gateway_ip>` with your local host IP address.
-
-    **Linux**
-
-    ```bash
-    route add <destination_ip> gw <gateway_ip>
-    ```
-
-    **Windows**
-
-    ```cmd
-    route add <destination_ip> <gateway_ip>
-    ```
-
-7. In the console window with the running sample, press D to download the sample file and verify that it comes from secondary storage.
-
-8. Remove the invalid static route.
-
-    **Linux**
-
-    ```bash
-    route del <destination_ip> gw <gateway_ip>
-    ```
-
-    **Windows**
-
-    ```cmd
-    route delete <destination_ip>
-    ```
+If you don't have a Microsoft Azure subscription, you can get a FREE trial account <a href="http://go.microsoft.com/fwlink/?LinkId=330212">here</a>.
 
-9. In the console window with the running sample, press D to download the sample file and verify that it comes from primary storage. 
+# Folders introduction
 
-10. Press Q to quit the sample and delete the container and file from storage.
+Two folders are referred to different version of Azure SDK.
```

Comment:
Please turn this into a sentence like: You will find the following folders: key-vault-node-quickstart-3, which references the X.X SDK and keyvault-node-quickstart-v4, which uses the X.X version of the SDK.

Something like that.